services/web-app/components/hero/hero.module.scss

```diff
@@ -56,18 +57,36 @@ $illo-original-height: 1078px;
   @include type-style('body-long-02', true);
 }
 
-.image-wrap {
+.image-container {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  overflow: hidden;
+  width: 100vw;
+  height: $illo-mobile-height;
+
+  @include breakpoint(md) {
+    top: 0;
+    bottom: auto;
+    height: $hero-height;
+  }
+}
+
+.image-position {
   position: absolute;
   top: 0;
   right: -25vw;
-  overflow: hidden;
   width: 50%;
   height: $hero-height;
   @include breakpoint(xlg) {
     // -18vw chosen to horizontally center the illo in white space at ultra max viewport
     right: -18vw;
     width: $hero-height * ($illo-original-width / $illo-original-height);
   }
+  @include breakpoint(max) {
+    // match xlg position
+    right: -285px;
```

Comment:
not that it matters too much but could we so some version of this with tokens?  this below equates to 17.5 rem vs.  285px. (17.813 rem) or does it have to be exactly 285px.  

I know this is a very small details -  was just curious 😊

Reply:
@andreancardona this is positioning the background SVG so tokens don't make a lot of sense here. We're taking the generic SVG, and scaling and positioning it to match the spec. Because I couldn't use tokens here, I added CSS comments to explain the arbitrary values.